lib/alephant/broker/cache/cached_object.rb

```diff
@@ -11,41 +11,39 @@ class CachedObject
         DEFAULT_TTL = 10
 
         def initialize(obj)
-          logger.info(event:   'SettingCachedObject',
+          logger.debug(event:  'SettingCachedObject',
                       content: obj,
                       method:  "#{self.class}#initialize")
-
           @s3_obj = obj
         end
 
         def update(obj)
-          logger.info(event:       'UpdatingCachedObject',
+          logger.debug(event:      'UpdatingCachedObject',
                       old_content: @s3_obj,
                       new_content: obj,
                       method:      "#{self.class}#update")
-
           @s3_obj = obj
         end
 
         def updated
           time = metadata[:'head_Last-Modified']
           Time.parse(time)
         rescue TypeError, ArgumentError => error
-          logger.error(method: "#{self.class}#updated", error: error)
+          logger.error(event: 'updateError', method: "#{self.class}#updated", error: error)
           Time.now
         end
 
         def ttl
           Integer(metadata[:ttl] || metadata['ttl'])
         rescue TypeError => error
-          logger.error(method: "#{self.class}#ttl", error: error)
+          logger.error(event: 'ttlError', method: "#{self.class}#ttl", error: error)
```

Comment:
`ttlError` might still be a bit vague although it is technically correct I suppose. `NonIntegerTTLError` or something similar might be more useful